src/index.js

```diff
@@ -1,17 +1,39 @@
 /* global projectName */
-/* eslint no-unused-vars: ["error", { "varsIgnorePattern": "[iI]gnored" }]*/
+
+/*
+ This file dynamically generates the user interface for the freeCodeCamp
+ testable-projects application. The user interface consists of two main parts:
+ 1. An <iframe> situated in the top-left corner of the viewport with controls
+    for running the tests (loaded into document at #fcc_test_suite_wrapper),
+    as well as a small read-only indicator in the top-right corner of the
+    viewport that shows the pending test project at
+    #fcc_test_suite_indicator_wrapper
+ 2. A modal <div id="mocha"> automatically inserted into the document via Mocha
+
+ The test controller window runs in an iframe in order to encapsulate the DOM
+ elements and avoid interfering with student project code.
+ Since Mocha cannot automatically insert its <div id="mocha"> into the iframe,
+ we need to append it and load its styles separately.
+
+ We can use Webpack to inject loaded css into the document, but we append a
+ <style> element with a string of css loaded (not injected) by Webpack via
+ jQuery in the iframe. Therefore, we maintain two css files:
+ one for the test control window (test-suite.css), and one for the Mocha modal
+ (modal.css)
+*/
 
 import $ from 'jquery';
 import chai from 'chai';
 // Webpack is configured to load those files with the .html extension as Strings
 import testSuiteSkeleton from './utils/test-suite-skeleton.html';
-// the !- prefixes are for process arguments respective of plugins
-// Example: https://stackoverflow.com/a/42440360/3530394
-// style-loader injects css loaded by css-loader through this import statement.
-
+import modalSkeleton from './utils/modal-skeleton.html';
+// Test window-specific css loaded by css-loader through webpack config only.
+import testSuiteFCCStyles from './stylesheets/test-suite.css';
+// style-loader injects css from css-loader into document
+// Using inline loading of style-loader in order to only inject modal css
 /* eslint import/no-unresolved: [2, { ignore: ['!style-loader.*$'] }] */
-import testSuiteFCCStylesIgnored from
-  '!style-loader!css-loader!./stylesheets/style.css';
+import modalFCCStyles from // eslint-disable-line no-unused-vars
+  '!style-loader!css-loader!./stylesheets/modal.css';
```

Comment:
These names are really confusing. I think `modal.css` is for Mocha? But the variable name we are using has 'FCC' in it? If we can be consistent with the names it will help. Maybe we need to come up with 2 prefixes that are always used consistently in both file names and variables? Something like 'mochaModal' (or mocha-modal for file names) and 'fCC-test-UI' (fcc-test-ui for file names)?